src/main/java/com/car/app/tests/AllTests.java

```diff
@@ -0,0 +1,13 @@
+package com.car.app.tests;
+
+
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+
+@RunWith(Suite.class)
+@SuiteClasses({CarServiceTest.class })
+public class AllTests {
```

Comment:
What is the need of this class ?

Reply:
in case we have a lot of classes, so to run all of them in same time, we use this class for that,
for example to test CarService and CarDao in same time, we'll use this class

Reply:
![image](https://user-images.githubusercontent.com/78147936/107245022-f9633380-6a2e-11eb-886a-0d3e0adcf4bd.png)